### DISCUSSION TITLE
2 suggestions for 86Box

### DISCUSSION THREAD

**Discussion by smfthe1199** [86Box/86Box#4602] [Ideas]
All I can request is the monitor selection in the "Video" tab in the 86Box configuration thing.

In addition, I want the 86Box Manager (86Manager) to be advanced. Thank you :).

**Comment by smfthe1199** [comment]
Anybody not answering -_-

Oh well.

**Reply by TC1995** [reply]
You can't expect us to be 24/7 operating.

**Reply by smfthe1199** [reply]
Well I'm still waiting what do you expect?

**Reply by TC1995** [reply]
Still we have lives to deal with.

**Reply by smfthe1199** [reply]
I forgot about this word but I am aware..
